source/geometry/IntBox2D.ooc

```diff
@@ -102,6 +102,8 @@ IntBox2D: cover {
 		this left <= point x && point x < this right && this top <= point y && point y < this bottom
 	}
 	contains: func ~box (box: This) -> Bool { this intersection(box) == box }
+	toFloatBox2D: func -> FloatBox2D { FloatBox2D new(this left, this top, this width, this height) }
```

Comment:
Is there an implicit cast `as` that you can overload for `FloatBox2D`?
like `floatbox := myIntBox as FloatBox2D`?